tests/Microsoft.Identity.Test.Core.UIAutomation/CoreUiTestConstants.cs

```diff
@@ -54,21 +48,26 @@ public static class CoreUiTestConstants
         public const string TestResultSuccessfulMessage = "Result: Success";
         public const string TestResultFailureMessage = "Result: Failure";
 
-        //MSAL test app
         public const string DefaultScope = "User.Read";
         public const string AcquirePageID = "Acquire";
         public const string CachePageID = "Cache";
         public const string SettingsPageID = "Settings";
         public const string ScopesEntryID = "scopesList";
         public const string UiBehaviorPickerID = "uiBehavior";
         public const string SelectUser = "userList";
-        public const string UserNotSelected = "not selected";
         public const string UserMissingFromResponse = "Missing from the token response";
         public const string RedirectUriOnAndroid = "urn:ietf:wg:oauth:2.0:oob";
-        public const string RedirectUriEntryID = "redirectUriEntry";
+
+        //Consent page
+        public const string ConsentHeader = "consentHeader";
+        public const string PermissionsRequested = "Permissions requested";
+        //Third party consent page
+        public const string AppHeader = "idSpan_AppHeaderText";
+        public const string AppAccessTitle = "Let this app access your info?";
+        public const string WebConsentSubmitId = "idBtn_Accept";
+        public const string MsaUserHomeUpn = "MSIDLAB4@outlook.com";
```

Comment:
Shouldn't we get this from the lab?

Reply:
only for a third party app will you get a consent screen, so i need to check for a specific user.

---
In reply to: [241014453](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/721#discussion_r241014453) [](ancestors = 241014453)

Reply:
probably there should exist other parameters such as "third party app" which should be settable in the query API?

---
In reply to: [241241286](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/721#discussion_r241241286) [](ancestors = 241241286,241014453)

Reply:
we can also table this and i can work w/lab team to get a more concise query and go from there. we're the first to use msa accounts, so i think it's a little rough still. 

---
In reply to: [241243168](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/721#discussion_r241243168) [](ancestors = 241243168,241241286,241014453)